scripts/ingests/filip_15.py

```diff
@@ -0,0 +1,76 @@
+# script to get the data from table 9 of Filippazzo 2015 paper for modeled parameters
+
+from scripts.ingests.ingest_utils import *
+from scripts.ingests.utils import *
+from astropy.table import Table
+import astropy.units as u
+from astropy.coordinates import Angle
+
+SAVE_DB = False  # save the data files in addition to modifying the .db file
+RECREATE_DB = True  # recreates the .db file from the data files
+VERBOSE = False
+# LOAD THE DATABASE
+db = load_simpledb('SIMPLE.db', recreatedb=RECREATE_DB)
+
+# WRITE THE JSON FILES
+if SAVE_DB:
+    db.save_database(directory='data/')
+# live google sheet
+link = "https://docs.google.com/spreadsheets/d/e/2PACX-1vTJZccUnKGOLcfPixaS3Tib0sqqC23n1Q_0ybjwXN4ZDYBT_-2_FeHLShVBzkIDfo39-tV1aoz5rCth/pub?output=csv"
+
+columns = ['Designation 9', 'pi', 'pi References', 'log(L_*/L_sun)', 'Age^a', 'Radius', 'log(g)', 'T_eff', 'Mass',
+           'R.A.', 'Decl.', 'Designation 1', 'Discovery References', 'SpT', 'SpT References']
+filip15_table = Table.read(link, format='ascii', data_start=6, data_end=203, header_start=4, names=columns, guess=False,
+                           fast_reader=False, delimiter=',')
+
+data_columns = ['Radius', 'log(g)', 'T_eff', 'Mass']  # columns with wanted data values
+
+for row in filip15_table:  # rewriting so that the empty values are None
+    for column in data_columns:
+        if row[column] == 'cdots':
+            row[column] = None
```

Comment:
rather than looping through each row, you can use the `np.where` function to find the "cdots". In general, you want to avoid loops,  where possible.
```suggestion
# replacing empty values ('cdots') with None
    for column in data_columns: 
        filip15_table[column][np.where(filip15_table[column] == 'cdots')] = None
```
The links that I used to help figure this out:
- https://docs.astropy.org/en/stable/table/modify_table.html
- https://stackoverflow.com/questions/16642022/numpy-where-on-string

Reply:
I also just realized you could do this in your `get_float_value` function.